index.bs

```diff
@@ -653,6 +659,51 @@ well as to decode and validate the bindings of both the client and authenticator
 The scoped credential type uses certain data structures that are specified in supporting specifications. These are as follows.
 
 
+### Client data used in WebAuthn signatures (dictionary <dfn dictionary>ClientData</dfn>) ### {#sec-client-data}
+
+The client data represents the contextual bindings of both the [RP] and the client platform. It is a key-value mapping with
+string-valued keys. Values may be any type that has a valid encoding in JSON. Its structure is defined by the following Web IDL.
+
+<pre class="idl">
+    dictionary ClientData {
+        required DOMString           challenge;
+        required DOMString           origin;
+        required DOMString           rpId;
```

Comment:
I think the rpId is also signed over outside of the clientData (see bytes 0-31 in the Authenticator Data part of the signature format).  I think we should consider removing rpId from the client data -- what do you think?